scripts/news/utils.py

```diff
@@ -0,0 +1,196 @@
+    MD5 hash of the text, converting it to base64 (using the "urlsafe" alphabet), and taking the first
+    6 characters of that.
+    """
+    digest = hashlib.md5(body.encode("utf-8")).digest()  # noqa: S303
+    return base64.urlsafe_b64encode(digest)[0:6].decode("ascii")
+
+
+def _out(message: Optional[str] = None, nl: bool = True, **styles: Any) -> None:
+    if message is not None:
+        if "bold" not in styles:
+            styles["bold"] = True
+        message = style(message, **styles)
+    echo(message, nl=nl, err=True)
+
+
+def _err(message: Optional[str] = None, nl: bool = True, **styles: Any) -> None:
+    if message is not None:
+        if "fg" not in styles:
+            styles["fg"] = "red"
+        message = style(message, **styles)
+    echo(message, nl=nl, err=True)
+
+
+def out(message: Optional[str] = None, nl: bool = True, **styles: Any) -> None:
+    """Utility function to output a styled message to console."""
+    _out(message, nl=nl, **styles)
+
+
+def err(message: Optional[str] = None, nl: bool = True, **styles: Any) -> None:
+    """Utility function to output a styled error message to console."""
+    _err(message, nl=nl, **styles)
+
+
+class NotRequiredIf(Option):
+    """Custom option class to make option mutually exclusive with another i.e. 'not_required_if'."""
+
+    def __init__(self, *args, **kwargs):
+        self.not_required_if = kwargs.pop("not_required_if")
+        assert self.not_required_if, "'not_required_if' parameter required"  # noqa: S101
+        kwargs["help"] = (
+            kwargs.get("help", "")
+            + " NOTE: This argument is mutually exclusive with %s" % self.not_required_if
+        ).strip()
+        super(NotRequiredIf, self).__init__(*args, **kwargs)
+
+    def handle_parse_result(
+        self, ctx: Context, opts: Mapping[str, Any], args: List[str]
+    ) -> Tuple[Any, List[str]]:
+        """Check if option is mutually exclusive with another, if yes print error and exist."""
+        other_present = self.not_required_if in opts
+
+        if other_present:
+            we_are_present = self.name in opts
+            if we_are_present:
+                err(
+                    f"{ERROR_MSG_PREFIX} Illegal usage. `%s` is mutually exclusive with `%s`"
+                    % (self.name, self.not_required_if),
+                    fg="red",
+                )
+                ctx.exit(code=1)
+            else:
+                self.prompt = None
+
+        return super(NotRequiredIf, self).handle_parse_result(ctx, opts, args)
+
+
+def sanitize_section(section: str) -> str:
+    """Cleans up a section string, making it viable as a directory name."""
+    return section.replace("/", "-").lower()
+
+
+def glob_fragments(version: str, sections: List[str]) -> List[str]:
+    """Glob all news fragments present on the repo."""
+    filenames = []
+    base = os.path.join("news", version)
+
+    if version != "next":
```

Comment:
`if version.lower() != "next":`

Reply:
`NEXT` could be passed, we shouldn't assume that there was validation beforehand